js/src/extension.ts

```diff
@@ -2,19 +2,25 @@
 // It contains some requirejs configuration and the `load_ipython_extension`
 // which is required for any notebook extension.
 
+import * as Handsontable from 'handsontable/dist/handsontable.full.js';
+import 'handsontable/dist/handsontable.min.css';
+import '../../src/custom.css';
+
```

Comment:
This will give an issue with the renderer extension, since you want only 1 handsontable (both extensions the same) we need to make it into a separate module.

Reply:
I have only one `handsontable`, this one :)